src/main/java/mate/academy/bookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.bookstore.validator.FieldMatch;
+
+@Data
+@FieldMatch(
+        field = "password",
+        fieldMatch = "repeatPassword",
+        message = "Passwords does not match")
+public class UserRegistrationRequestDto {
+    private static final String EMAIL_VALIDATION_REGEX
+            = "^((?!\\.)[\\w\\-_.]*[^.])(@\\w+)(\\.\\w+(\\.\\w+)?[^.\\W])$";
+
+    @Email(regexp = EMAIL_VALIDATION_REGEX)
```

Comment:
is it necessary to add a custom regex?

Reply:
I couldn't find any examples of valid emails for standard validation... so I searched for a customized regex.